src/base/utils/net.h

```diff
@@ -28,13 +28,26 @@
 
 #ifndef BASE_UTILS_NET_H
 #define BASE_UTILS_NET_H
+
+#include <QList>
+
+class QHostAddress;
 class QString;
+class QStringList;
+template <typename T, typename U> struct QPair;
 
 namespace Utils
 {
     namespace Net
     {
+        using Subnet = QPair<QHostAddress, int>;
+
         bool isValidIP(const QString &ip);
+        bool parseSubnet(const QString &subnetStr);
```

Comment:
Since this function does not give us the parsed subnet, I would name it `canParseSubnet`. IMO, use overloads is incorrect in this case.